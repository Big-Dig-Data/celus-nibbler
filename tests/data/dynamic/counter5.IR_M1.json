{
  "kind": "counter5.IR_M1",
  "parser_name": "MY_IR_M1_CUSTOM",
  "version": 1,
  "areas": [
     {
        "kind": "counter5.IR_M1"
     }
  ]
}
