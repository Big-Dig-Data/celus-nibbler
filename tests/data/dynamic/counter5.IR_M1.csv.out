../counter/5/IR_M1-a.csv.out